core/unit_test/tools/TestTuning.cpp

```diff
@@ -88,16 +88,16 @@ int main() {
     // test that ID's are transmitted to the tool
     Kokkos::Tools::Experimental::set_declare_output_type_callback(
         [](const char*, const size_t,
-           Kokkos::Tools::Experimental::VariableInfo& info) {
-          if (info.type !=
+           Kokkos::Tools::Experimental::VariableInfo* info) {
+          if (info->type !=
               Kokkos::Tools::Experimental::ValueType::kokkos_value_int64) {
             throw(std::runtime_error("Tuning Variable has wrong type"));
           }
         });
     Kokkos::Tools::Experimental::set_declare_input_type_callback(
         [](const char*, const size_t,
-           Kokkos::Tools::Experimental::VariableInfo& info) {
-          if (info.type !=
+           Kokkos::Tools::Experimental::VariableInfo* info) {
+          if (info->type !=
```

Comment:
How did that compile before?

Reply:
It didn't, I didn't realize we didn't have Tuning enabled in Jenkins. Now that Daniel helped me out we catch these bugs